src/test/python/sqlalchemy/all_types.py

```diff
@@ -0,0 +1,36 @@
+from sqlalchemy import Column, Integer, String, Boolean, LargeBinary, Float,\
+  Numeric, DateTime, Date
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.orm import registry
+
+mapper_registry = registry()
+Base = mapper_registry.generate_base()
+
+
+class AllTypes(Base):
+  __tablename__ = "all_types"
+
+  col_bigint = Column(Integer, primary_key=True)
+  col_bool = Column(Boolean)
+  col_bytea = Column(LargeBinary)
+  col_float8 = Column(Float)
+  col_int = Column(Integer)
+  col_numeric = Column(Numeric)
+  col_timestamptz = Column(DateTime(timezone=True))
+  col_date = Column(Date)
+  col_varchar = Column(String)
+  col_jsonb = Column(JSONB)
+
+  def __repr__(self):
```

Comment:
Does SQLAlchemy really not have a builtin that does this?

... apparently maybe it doesn't?  I see an extension that adds it:  https://pypi.org/project/sqlalchemy-repr/

Reply:
I couldn't find any builtin function for this and retrofitted an example I found in a tutorial online somewhere (cannot find it . And so far we have been trying to keep our ORM samples as native as possible, i.e. not add any handy extensions unless there's no other choice (one exception to this is for example that we added an extra library to the Hibernate sample to be able to support `jsonb`, as Hibernate itself does not have native support for it).